tests/sources/xml/test_oai_dc.py

```diff
@@ -54,26 +65,217 @@ def test_oaidctransform_with_all_fields_transforms_correctly():
 
 def test_oaidc_transform_with_optional_fields_blank_transforms_correctly():
     source_records = OaiDc.parse_source_file(
-        f"{FIXTURES_PREFIX}/oaidc_record_optional_fields_blank.xml"
+        "tests/fixtures/oai_dc/oaidc_record_optional_fields_blank.xml"
+    )
+    output_records = OaiDc("cool-repo", source_records)
+    assert next(output_records) == timdex.TimdexRecord(
+        source="A Cool Repository",
+        source_link="https://example.com/guides/175846",
+        timdex_record_id="cool-repo:guides-175846",
+        title="Materials Science & Engineering",
+        citation="Materials Science & Engineering. cool-repo. "
+        "https://example.com/guides/175846",
+        content_type=["cool-repo"],
+        format="electronic resource",
+        identifiers=[
+            timdex.Identifier(value="oai:libguides.com:guides/175846", kind="OAI-PMH")
+        ],
     )
-    output_records = OaiDc("libguides", source_records)
-    assert next(output_records) == BLANK_OR_MISSING_OPTIONAL_FIELDS_TIMDEX
 
 
-def test_oaidc_transform_with_optional_fields_missing_transforms_correctly():
+def test_oai_dc_transform_with_optional_fields_missing_transforms_correctly():
     source_records = OaiDc.parse_source_file(
-        f"{FIXTURES_PREFIX}/oaidc_record_optional_fields_missing.xml"
+        "tests/fixtures/oai_dc/oaidc_record_optional_fields_missing.xml"
+    )
+    output_records = OaiDc("cool-repo", source_records)
+    assert next(output_records) == timdex.TimdexRecord(
+        source="A Cool Repository",
+        source_link="https://example.com/guides/175846",
+        timdex_record_id="cool-repo:guides-175846",
+        title="Materials Science & Engineering",
+        citation="Materials Science & Engineering. cool-repo. "
+        "https://example.com/guides/175846",
+        content_type=["cool-repo"],
+        format="electronic resource",
+        identifiers=[
+            timdex.Identifier(value="oai:libguides.com:guides/175846", kind="OAI-PMH")
+        ],
     )
-    output_records = OaiDc("libguides", source_records)
-    assert next(output_records) == BLANK_OR_MISSING_OPTIONAL_FIELDS_TIMDEX
 
 
-def test_oaidc_generic_date():
+def test_get_content_type_success(oai_dc_record_all_fields):
     source_records = OaiDc.parse_source_file(
-        f"{FIXTURES_PREFIX}/oaidc_record_valid_generic_date.xml"
+        "tests/fixtures/oai_dc/oaidc_record_all_fields.xml"
     )
-    transformer_instance = OaiDc("libguides", source_records)
-    xml = next(transformer_instance.source_records)
-    assert transformer_instance.get_dates("test_source_record_id", xml) == [
-        timdex.Date(kind="Unknown", note=None, range=None, value="2008-06-19T17:55:27")
+    assert OaiDc("cool-repo", source_records).get_content_type() == ["cool-repo"]
```

Comment:
Because the `content_type` is derived from the `source` instance variable, `OaiDc` must be instantiated for the test.